es/_posts/2023-08-01-047.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: '#047 git clone -b nombre'
```

Comment:
The title is the big letters, and sub-title and pre-title all the text around the big letters.

This is the Portuguese version below:

<img width="852" height="279" alt="Image" src="https://github.com/user-attachments/assets/b5890906-aa14-45e8-a780-7bf0a89b9eb7" />

it would have:

```
pretitle: fazendo
title: clone
subtitle: de um branch específico
```

And this is the English version:
<img width="732" height="247" alt="Image" src="https://github.com/user-attachments/assets/2a35da52-4b2f-45c2-be33-df341f539487" />

it would have:

```
title: Clone
subtitle: from a specific branch
```

please adjust the translation accordingly thanks.